src/main/kotlin/io/bazel/kotlin/builder/tasks/jvm/CompilationArgs.kt

```diff
@@ -172,28 +169,18 @@ class CompilationArgs(
     return this
   }
 
-  fun list(): List<String> = args.toList()
-
-  fun base64Encode(
+  fun flagRepeated(
     flag: String,
     vararg values: Pair<String, List<String>>,
-    transform: (String) -> String = { it }
+    transform: (option: String, value: String) -> String
   ): CompilationArgs {
-    val os = ByteArrayOutputStream()
-    val oos = ObjectOutputStream(os)
-
-    oos.writeInt(values.size)
-    for ((k, vs) in values) {
-      oos.writeUTF(k)
-
-      oos.writeInt(vs.size)
-      for (v in vs) {
-        oos.writeUTF(v)
+    values.forEach { (option, values) ->
+      values.forEach {
+        flag(flag, transform(option, it))
       }
     }
-
-    oos.flush()
-    flag(flag, transform(Base64.getEncoder().encodeToString(os.toByteArray())))
     return this
   }
+
+  fun list(): List<String> = args.toList()
```

Comment:
Probably should stick with the toList() naming pattern.

Reply:
Can't just merge it as we needed the base64 encoding for older versions of kotlinc. Currently working on a way to preserve versionsing without making too much of a mess.

Reply:
Added branch to cover pre-1.5

Not the way I would like it, but I don't want to hold up progress while I yak shave with one hand.